guides/embedding_gradio_demos.md

```diff
@@ -10,48 +10,45 @@ Once you have created a Gradio demo and [hosted it on Hugging Face Spaces](https
+* (Preferred) **Web components**, using the `<gradio-app>` tag
 
-* **Iframes**, using the \<iframe\> tag, in cases where you cannot add javascript to your page
+* **Iframes**, using the `<iframe>` tag, in cases where you cannot add javascript to your page
 
-In either case, the first step is to create your Gradio demo and host it on Hugging Face Spaces, a process which is described in this blog post: https://huggingface.co/blog/gradio-spaces 
+In either case, the first step is to create your Gradio demo and host it on Hugging Face Spaces, a process which is described in this blog post: [https://huggingface.co/blog/gradio-spaces](https://huggingface.co/blog/gradio-spaces) 
 
-In this guide, we will be embedding this image classification Space: https://huggingface.co/spaces/abidlabs/pytorch-image-classifier using both web components and  
+In this guide, we will be embedding this image classification Space: [https://huggingface.co/spaces/abidlabs/pytorch-image-classifier](https://huggingface.co/spaces/abidlabs/pytorch-image-classifier) using both web components and iframes.
 
 ### Embedding with Web Components 
 
 Embedding with web components has a number of advantages over traditional iframes:
+
 * It's easy! All you need is the name of the Space
 * It's faster to load than iframes 
 * The embedded demo takes up just the right width and height for the demo
 
 How do you embed your Gradio demo with web components? There are two steps:
 
-1. Somewhere in your webpage, usually in the \<head\> tags, include the following javascript code, which imports the Gradio package:
+1. Somewhere in your webpage, usually in the `<head>` tags, include the following javascript code, which imports the Gradio package:
 
-```html
-<script type="module" src="https://gradio.s3-us-west-2.amazonaws.com/3.0.18/gradio.js"></script>
-```
 
-* Instead of <code>3.0.18</code>, you can put whichever version of `gradio`, you'd like to run (a safe choice is the [latest stable version of Gradio](https://pypi.org/project/gradio/)). 
+    `<script type="module" src="https://gradio.s3-us-west-2.amazonaws.com/3.0.18/gradio.js"></script>`
+
+
+    * Instead of <code>3.0.18</code>, you can put whichever version of `gradio`, you'd like to run (a safe choice is the [latest stable version of Gradio](https://pypi.org/project/gradio/)). 
 
 2. Wherever you'd like to put your demo on your webpage, insert the following code:
 
-```html
-<gradio-app space="abidlabs/pytorch-image-classifier"> </gradio-app>
-```
+    `<gradio-app space="abidlabs/pytorch-image-classifier"> </gradio-app>`
 
-* Instead of <code>abidlabs/pytorch-image-classifier</code>, you should, of course, put the path to your Space
+    * Instead of <code>abidlabs/pytorch-image-classifier</code>, you should, of course, put the path to your Space
 
 That's it! Here's what it looks like in action:
 
 <gradio-app space="abidlabs/pytorch-image-classifier"> </gradio-app>
```

Comment:
Can someone else please verify that this demo is not showing up on the already published guide on https://gradio.app ? 

![image](https://user-images.githubusercontent.com/41651716/178841986-16fc6cfb-df8e-4a0a-b736-6276a9065312.png)

Reply:
Hmm I refreshed the page, sometimes it worked and sometimes it didn't. Something flaky going on here.